### PR TITLE
Update HTTPClient.md

### PR DESCRIPTION
**Pull request by riccardozambito**
Very important missing information added: the format of the response of HTTP methods.
@davidebianchi  are these changes correct? Are they valid for direct and non-direct service proxies both?

#### Checklist

- [x] your branch will not cause merge conflict with `master`
- [x] run `npm test`
- [x] tests are included
- [x] the documentation is updated or intregrated accordingly with your changes
- [x] the code will follow the lint rules and style of the project
- [x] you are not committing extraneous files or sensible data

### CODE REVIEW

File: docs/HTTPClient.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ Both proxies expose the methods to perform a specific HTTP request to service.
  * `patch(path, body, querystring, options)`
  * `delete(path, body, querystring, options)`
 
+All methods return a *Promise object*. You can access to response body trough the `payload` property of the returned object.
```
Comment (davidebianchi) [inline docs/HTTPClient.md:30]: If we write how to access to the body, we should also explain how to access to other fields of the response such as headers. What do you think?

Comment (riccardozambito) [reply docs/HTTPClient.md:30]: Yes, we should, Can we access to headers with `payload.headers`?

Comment (davidebianchi) [reply docs/HTTPClient.md:30]: Yes, headers and statusCode

Comment (riccardozambito) [reply docs/HTTPClient.md]: @davidebianchi I have updated the file

**Review by davidebianchi — APPROVED** [review]

**Pull request merged**